spaceship/lib/spaceship/client.rb

```diff
@@ -186,6 +186,54 @@ def team_id=(team_id)
       @current_team_id = team_id
     end
 
+    # @return (String) The currently selected Team Name
+    def team_name
+      return @current_team_name if @current_team_name
+
+      if teams.count > 1
+        puts("The current user is in #{teams.count} teams. Pass a team Name or call `select_team` to choose a team. Using the first one for now.")
+      end
+      @current_team_name ||= teams[0]['contentProvider']['name']
+    end
```

Comment:
@RishabhTayal Now that I think about this, we should probably 💣 `@current_team_name` and replace it as a method that looks up from `teams` based on `@current_team_id`